concepts/language-model.md

```diff
@@ -0,0 +1,80 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model takes text input (usually a history) and outputs the next word or character.
+
+Example:
+- input: `the man is riding a ___`
+- output: `bike` 40%, `car`: 30%, `scooter`: 10%, `blue`: 0.5%, ...
+
+In this example, the word `bike` has a 40% probability of being the continuation and is the most likely.
+
+## Decoding
+
+It is possible to do *decoding*, which means repeatedly taking e.g. the most probable output, adding it to the text and so on.
```

Comment:
We can introduce any concept x with *It is possible to do x...* but it's kinda just filler.

Maybe something like?
*Decoding is used to convert an abstract representation into a [string](string.md).*

Reply:
Updated.